src/shared/components/button/button.component.tsx

```diff
@@ -33,6 +42,7 @@ export const Button = ({
   const iconTypes = {
     [ButtonIcon.ARROW]: () => <ArrowIcon />,
     [ButtonIcon.OUT]: () => <OutIcon />,
+    [ButtonIcon.GET_IN_TOUCH]: () => <GetInTouchIcon />,
   };
```

Comment:
We could have some more scalable approach for icons in buttons, but for now I think it's good enough